packages/smooth_app/lib/pages/onboarding/consent_analytics_page.dart

```diff
@@ -0,0 +1,188 @@
+                  borderRadius: BorderRadius.circular(25.0),
+                  boxShadow: <BoxShadow>[
+                    BoxShadow(
+                        blurRadius: 3.0,
+                        color: shadowColor,
+                        offset: Offset(size.width * 0.004, size.height * 0.004))
+                  ]),
+              child: Row(
+                crossAxisAlignment: CrossAxisAlignment.center,
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  Text(
+                    appLocalizations.authorize_button_label,
+                    style: TextStyle(
+                        fontWeight: FontWeight.bold,
+                        color: Colors.white,
+                        fontSize: size.height * 0.025),
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(left: size.width * 0.02),
+                    child: Icon(
+                      Icons.check,
+                      color: Colors.white,
+                      size: size.height * 0.04,
+                    ),
+                  )
+                ],
+              ),
+            ),
+          ),
+
+          SizedBox(
+            height: size.height * 0.02,
+          ),
+
+          //Refuse Button
+          InkWell(
+            borderRadius: BorderRadius.circular(25.0),
+            onTap: () {
+              _analyticsLogic(false, userPreferences, localDatabase, context);
+            },
+            child: Ink(
+              height: size.height * 0.06,
+              width: size.width * 0.7,
+              decoration: BoxDecoration(
+                  color: Colors.red,
+                  borderRadius: BorderRadius.circular(25.0),
+                  boxShadow: <BoxShadow>[
+                    BoxShadow(
+                      blurRadius: 3.0,
+                      color: shadowColor,
+                      offset: Offset(size.width * 0.004, size.height * 0.004),
+                    )
+                  ]),
+              child: Row(
+                crossAxisAlignment: CrossAxisAlignment.center,
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  Text(
+                    appLocalizations.refuse_button_label,
+                    style: TextStyle(
+                        fontWeight: FontWeight.bold,
+                        color: Colors.white,
+                        fontSize: size.height * 0.025),
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(left: size.width * 0.02),
+                    child: Icon(
+                      Icons.close,
+                      color: Colors.white,
+                      size: size.height * 0.04,
+                    ),
+                  )
+                ],
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  // ignore: avoid_void_async
```

Comment:
Cou can also make it a `Future<void>` then you don't need this ignore

Reply:
completed